web/themes/joinup/templates/user/user--group-header.html.twig

```diff
@@ -26,49 +26,69 @@
-      <div class="featured__logo">{{ content.field_user_photo }}</div>
-      <div class="featured__text featured__text--double">
-        <h1><span>{{ full_name }}</span></h1>
-        {% if content.field_user_business_title|render %}
-          <div class="featured__text-small">{{ content.field_user_business_title }}</div>
-        {% endif %}
-      </div>
-    </a>
-    {% if content.field_social_media|render %}
-      <div class="featured__container">
-        {{ content.field_social_media }}
-      </div>
-    {% endif %}
-      <div class="featured__stats">
-        <div class="stats__column">
-          <div class="stats__wrapper">
-            <span class="stats__text">{{ 'Member for'|t }}:</span>
-            <span class="stats__text bold">{{ member_for }}</span>
+    <div class="mdl-grid mdl-grid--no-spacing">
+      <div class="mdl-cell--8-col mdl-cell--8-col-tablet mdl-cell--2-offset-desktop">
+
+        <div class="mdl-grid">
+
+          <div class="mdl-cell mdl-cell--4-col mdl-cell--3-col-wide">
+            <a class="featured__logo-wrapper" href="{{ path('entity.user.canonical',{'user': user.id()}) }}">
+              <div class="featured__logo" style="background-image: url({{ field_user_photo_url }})"></div>
+            </a>
+            {% if content.field_social_media|render %}
+              {{ content.field_social_media }}
+            {% endif %}
           </div>
-          {% if content.field_user_nationality|render %}
-            <div class="stats__wrapper">
-              <span class="stats__text">{{ 'Country of origin'|t }}:</span>
-              <span class="stats__text bold">{{ content.field_user_nationality }}</span>
+
+          <div class="mdl-cell mdl-cell--8-col mdl-cell--9-col-wide">
+            <div class="featured__text featured__text--double">
+              <h1><span>{{ full_name }}</span></h1>
+              {% if content.field_user_business_title|render %}
+                <div class="featured__text-small bold">{{ content.field_user_business_title }}</div>
+              {% endif %}
             </div>
-          {% endif %}
-          {% if content.field_user_organisation|render %}
-            <div class="stats__wrapper">
-              <span class="stats__text">{{ 'Organisation'|t }}:</span>
-              <span class="stats__text bold">{{ content.field_user_organisation }}</span>
+
+            <div class="mdl-grid mdl-grid--no-spacing">
+
+              <div class="mdl-cell mdl-cell--6-col mdl-cell--4-col-wide">
+                {% if content.field_user_organisation|render %}
+                  <div class="stats__wrapper stats__wrapper-organisation">
+                    <span class="stats__text bold">{{ 'Organisation'|t }}:</span>
+                    <span class="stats__text">{{ content.field_user_organisation }}</span>
+                  </div>
+                {% endif %}
+              </div>
```

Comment:
If the user doesn't enter an organisation then there is a strange void left in place here. We should probably not render the empty div in this case.

![2020-04-06-115304_7680x2160_scrot](https://user-images.githubusercontent.com/442924/78541145-d4de6f80-77fd-11ea-9109-e6f2b8dc8cb5.png)

Reply:
Thank you for this feedback,PR updated accordingly.